CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Changelog
 master
 -----
 
+3.3.0
+-----
+- added `animate` operator on UIViewPropertyAnimator
```

Comment:
What do you think about changing both of these to `Added UIViewPropertyAnimator Reactive Extensions`?

Reply:
done.